airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/config/UploaderConfig.java

```diff
@@ -7,44 +7,20 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.cloud.bigquery.BigQuery;
 import io.airbyte.integrations.base.destination.typing_deduping.StreamConfig;
-import io.airbyte.integrations.destination.bigquery.BigQueryUtils;
-import io.airbyte.integrations.destination.bigquery.UploadingMethod;
 import io.airbyte.integrations.destination.bigquery.formatter.BigQueryRecordFormatter;
-import io.airbyte.integrations.destination.bigquery.uploader.UploaderType;
 import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream;
-import java.util.Map;
 import lombok.Builder;
 import lombok.Getter;
 
 @Builder
 @Getter
 public class UploaderConfig {
 
-  private JsonNode config;
-  /**
-   * Taken directly from the {@link ConfiguredAirbyteStream}, except if the originalNamespace was
-   * null, we set it to the destination default originalNamespace.
-   */
-  private ConfiguredAirbyteStream configStream;
-  /**
-   * Parsed directly from {@link #configStream}.
-   */
+  private Integer bigQueryClientChunkSize;
+  private String datasetLocation;
   private StreamConfig parsedStream;
   private String targetTableName;
```

Comment:
how is this different from parsedStream.id.rawTableName?

Reply:
🪓  the class and call chain.